order/views.py

```diff
@@ -89,4 +90,22 @@ def dydx_order(self, request):
         except Exception as e:
             e = vars(e)
             result["error"] = e["msg"]["errors"][0]["msg"]
+            return Response(result, status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+    def orders(self, request):
+        result = {"message": None, "error": None}
+        request_data = request.query_params.dict()
+        self.serializer_class = FirestoreOrdersRequestSerializer(data=request_data)
+        self.serializer_class.is_valid(raise_exception=True)
```

Comment:
there is an issue we are using this api for getting all orders and order for an id so as you  use FirestoreOrdersRequestSerializer and you mark order_id as required  but when we want to get all the order not by i d then what we will pass and if we will not pass the id it will rise and exception .

Reply:
Done, made the change.

Reply:
did you test it properly  it is not working on my side 
can you share CURL for firestore all orders api

Reply:
run git pull --rebase